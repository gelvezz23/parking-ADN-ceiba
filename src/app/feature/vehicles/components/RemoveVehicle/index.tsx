import * as PropTypes from 'prop-types';
import * as React from 'react';
import { ButtonExit } from './styles';
import Modal from './../../../../shared/components/Modal';
import { Vehicle } from '../../models/Vehicle';

interface BtnRemoveVehicleProps {
  onClickMarkOut: (vehicle: Vehicle) => any;
  vehicle: Vehicle;
}

export const BtnRemoveVehicle: React.FC<BtnRemoveVehicleProps> = ({
  onClickMarkOut,
  vehicle,
}) => {
  const [modalIsOpen, setIsOpen] = React.useState(false);
  const OpenModal = () => {
    setIsOpen(true);
  };

  const CloseModal = () => {
    setIsOpen(false);
    //handleEliminar
  };

  const handleEliminar = () => onClickMarkOut(vehicle);
  return (
    <>
      <ButtonExit onClick={OpenModal}>
        <span role="img" aria-labelledby="trash">
          salida ðŸ•‘
        </span>
      </ButtonExit>

      {modalIsOpen && (
        <Modal
          vehicleModal={vehicle}
          onClose={CloseModal}
          onRemove={handleEliminar}
        />
      )}
    </>
  );
};

BtnRemoveVehicle.propTypes = {
  vehicle: PropTypes.shape({
    id: PropTypes.number.isRequired,
    day: PropTypes.string.isRequired,
    date: PropTypes.string.isRequired,
    isActive: PropTypes.bool.isRequired,
    slot: PropTypes.string.isRequired,
    responsable: PropTypes.string.isRequired,
    type: PropTypes.string.isRequired,
    idResponsable: PropTypes.string.isRequired,
    licensePlate: PropTypes.string.isRequired,
  }).isRequired,
  onClickMarkOut: PropTypes.func.isRequired,
};
