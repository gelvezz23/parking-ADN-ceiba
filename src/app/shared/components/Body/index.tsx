import styled from 'styled-components';

export const Body = styled.section`
  text-align: center;

  display: flex;
  justify-content: center;
  aling-items: center;

  @media (max-width: 600px) {
    flex-direction: column;
  }
`;
