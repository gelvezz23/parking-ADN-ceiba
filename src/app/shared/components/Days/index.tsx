export const Days = [
  'Lunes',
  'Martes',
  'Miercoles',
  'Jueves',
  'Viernes',
  'Sabado',
  'Domingo',
];
