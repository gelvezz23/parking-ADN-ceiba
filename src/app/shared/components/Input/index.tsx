import styled from 'styled-components';

export const Input = styled.input`
  border: 1px solid #ccc;
  border-radius: 25px;
  display: block;
  height: 50px;

  margin-bottom: 8px;
  padding: 10px 20px;
  width: 350px;

  &[disabled] {
    opacity: 0.3;
  }
  &:focus {
    outline: none !important;
    border-radius: 25px;
    border: 1px solid #2d6133;
  }

  @media (max-width: 600px) {
    width: 100%;
  }
`;
