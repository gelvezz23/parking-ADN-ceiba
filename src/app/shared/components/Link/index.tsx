import { Link as RouterLink } from 'react-router-dom';
import styled from 'styled-components';

export const Link = styled(RouterLink)`
  text-decoration: none;
  font-size: 20px;
  color: white;
  &:hover {
    color: rgba(0, 0, 0, 0.7);
  }

  @media (max-width: 1200px) {
    font-size: 15px;
  }
  @media (max-width: 600px) {
    font-size: 12px;
  }
`;
