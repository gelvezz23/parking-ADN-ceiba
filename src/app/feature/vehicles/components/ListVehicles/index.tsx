import * as PropTypes from 'prop-types';
import * as React from 'react';
import { BtnRemoveVehicle } from './../RemoveVehicle';
import { DivContainer } from './styles';
import { Slot } from './styles';
import { SlotInfo } from './styles';
import { Vehicle } from '../../models/Vehicle';

import imgCarro from './../../../../../assets/img/car.png';

import imgMoto from './../../../../../assets/img/moto.png';
import imgWeight from './../../../../../assets/img/weight.png';

export interface ListVehiclesProps {
  vehicles: Array<Vehicle>;
  onClickMarkOut: (vehicles: Vehicle) => void;
}

export const ListVehicles: React.FC<ListVehiclesProps> = ({
  vehicles,
  onClickMarkOut,
}) => {
  return (
    <>
      <h5>Motos</h5>
      <DivContainer>
        {vehicles.map((vehicle, index) => {
          return (
            vehicle.type === 'Moto' && (
              <Slot key={index}>
                <SlotInfo>
                  <img style={{ width: '50px' }} src={imgMoto} />
                  <p>{vehicle.responsable}</p>
                  <p>{vehicle.licensePlate}</p>
                  <p>{vehicle.type}</p>
                  <BtnRemoveVehicle
                    vehicle={vehicle}
                    onClickMarkOut={onClickMarkOut}
                  />
                </SlotInfo>
              </Slot>
            )
          );
        })}
      </DivContainer>

      <h5>Carros</h5>
      <DivContainer>
        {vehicles.map((vehicle, index) => {
          return (
            vehicle.type === 'Carro' && (
              <Slot key={index}>
                <SlotInfo>
                  <img style={{ width: '50px' }} src={imgCarro} />
                  <p>{vehicle.responsable}</p>
                  <p>{vehicle.licensePlate}</p>
                  <p>{vehicle.type}</p>
                  <BtnRemoveVehicle
                    vehicle={vehicle}
                    onClickMarkOut={onClickMarkOut}
                  />
                </SlotInfo>
              </Slot>
            )
          );
        })}
      </DivContainer>

      <h5>Vehiculos Pesados</h5>
      <DivContainer>
        {vehicles.map((vehicle, index) => {
          return (
            vehicle.type === 'Weight' && (
              <Slot key={index}>
                <SlotInfo>
                  <img style={{ width: '50px' }} src={imgWeight} />

                  <p>{vehicle.responsable}</p>
                  <p>{vehicle.licensePlate}</p>
                  <p>{vehicle.type}</p>
                  <BtnRemoveVehicle
                    vehicle={vehicle}
                    onClickMarkOut={onClickMarkOut}
                  />
                </SlotInfo>
              </Slot>
            )
          );
        })}
      </DivContainer>
    </>
  );
};

ListVehicles.propTypes = {
  vehicles: PropTypes.array.isRequired,
  onClickMarkOut: PropTypes.func.isRequired,
};
