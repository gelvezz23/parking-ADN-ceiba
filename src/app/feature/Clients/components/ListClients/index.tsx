import { Clients } from './../../models/Clients';
import { ContainerScrolling } from './styles';

import { DivContainer } from 'app/shared/components/Layout/styles';
import { Link } from './../../../../shared/components/Link';
import PropTypes from 'prop-types';
import React from 'react';
import { Table } from './styles';

interface ListClientsProps {
  clients: Array<Clients>;
}

const ListClients: React.FC<ListClientsProps> = ({ clients }) => {
  return (
    <DivContainer>
      <h5>Historial de Clientes</h5>
      <ContainerScrolling>
        <Table>
          <thead>
            <tr>
              <th>
                <h2>#</h2>
              </th>
              <th>
                <h2>Nombre</h2>
              </th>
              <th>
                <h2>Identificacion</h2>
              </th>
              <th>
                <h2>Placa de vehiculo</h2>
              </th>
              <th>
                <h2>Dia de registro</h2>
              </th>
              <th>
                <h2>Hora de registro</h2>
              </th>
            </tr>
          </thead>
          <tbody>
            {clients.map((client, index) => {
              return (
                <tr key={index}>
                  <td>
                    <h5>{client.id}</h5>
                  </td>
                  <td>
                    <h5>{client.responsable}</h5>
                  </td>
                  <td>
                    <h5>{client.idResponsable}</h5>
                  </td>
                  <td>
                    <h5>{client.licensePlate}</h5>
                  </td>
                  <td>
                    <h5>{client.day}</h5>
                  </td>
                  <td>
                    <h5>{`${new Date(client.date).getHours()}:${new Date(
                      client.date
                    ).getMinutes()}:${new Date(client.date).getSeconds()}`}</h5>
                  </td>
                </tr>
              );
            })}
          </tbody>
        </Table>
      </ContainerScrolling>
      <Link to="/">
        <h5>Volver</h5>
      </Link>
    </DivContainer>
  );
};
ListClients.propTypes = {
  clients: PropTypes.array.isRequired,
};

export default ListClients;
