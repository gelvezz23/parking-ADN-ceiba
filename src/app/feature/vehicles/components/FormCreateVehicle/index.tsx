import * as PropTypes from 'prop-types';
import * as React from 'react';
import * as Yup from 'yup';

import { Button } from '../../../../shared/components/Button';
import { CalculatestockCar } from './../../../../shared/utils/vehiclesStock/classStock/calculateStockCar';
import { CalculatestockMoto } from './../../../../shared/utils/vehiclesStock/classStock/calculateStockMoto';
import { CalculatestockWeight } from './../../../../shared/utils/vehiclesStock/classStock/calculateStockWeight';

import { Clients } from './../../../Clients/models/Clients';
import { FormikHelpers } from 'formik/dist/types';
import { Input } from '../../../../shared/components/Input';

import { Select } from '../../../../shared/components/Select';

import SpanErrorAlert from './../../../../shared/components/SpanError';
import { Vehicle } from '../../models/Vehicle';
import { getSundayOnWeek } from './../../../../shared/utils/getDaysOnWeek';
import { licensePlateReapet } from '../../../../shared/utils/LicensePlateReapet';
import { useFormik } from 'formik';

interface FormValues {
  slot: string;
  responsable: string;
  type: string;
  idResponsable: string;
  licensePlate: string;
}

interface FormCreateVehicleProp {
  onSubmit: (payload: Vehicle) => any;
  addClients: (payload: Clients) => any;
  disabled?: boolean;
  formTitle: string;
  vehicle: Array<Vehicle>;
  initialValues?: FormValues;
}

const validationSchema = Yup.object().shape<FormValues>({
  slot: Yup.string().required('Aqui debe ir la posicion de el carro'),
  responsable: Yup.string().required(
    'Debe ir el nombre de el due√±o de el vehiculo'
  ),
  type: Yup.string().required(
    'Que tipo de vehiculo es (moto, carro, vehiculo pesado)'
  ),
  idResponsable: Yup.string().required('Identificacion de el responsable'),
  licensePlate: Yup.string().required(
    'Debe ir placa de el vehiculo a registrar'
  ),
});

export const FormCreateVehicle: React.FC<FormCreateVehicleProp> = ({
  onSubmit,
  vehicle,
  addClients,
  disabled,
  formTitle,
  initialValues = {
    slot: '',
    day: '',
    responsable: '',
    type: '',
    idResponsable: '',
    licensePlate: '',
  },
}) => {
  const [error, setError] = React.useState('');

  const calculateStockMoto = new CalculatestockMoto();
  const calculateStockCar = new CalculatestockCar();
  const calculateStockWeight = new CalculatestockWeight();

  const permitionInsertStockMoto = calculateStockMoto.stockCalculate(vehicle);
  const permitionInsertStockCar = calculateStockCar.stockCalculate(vehicle);
  const permitionInsertStockWeight = calculateStockWeight.stockCalculate(
    vehicle
  );
  const sundayPosition = 6;
  const daysOnWeek = getSundayOnWeek(sundayPosition);

  const handleSubmit = (
    values: FormValues,
    { resetForm }: FormikHelpers<FormValues>
  ) => {
    const vehicleIsRepeat = licensePlateReapet(vehicle, values.licensePlate);

    if (vehicleIsRepeat) {
      setError('La placa de el vehiculo ya se encuentra registrada');
    }
    if (values.type === 'Moto') {
      if (permitionInsertStockMoto === false) {
        setError('El lugar de motos se encuentra lleno');
      }
      if (permitionInsertStockMoto === true && vehicleIsRepeat === false) {
        onSubmit({
          id: 0,
          day: daysOnWeek,
          date: new Date().toISOString(),
          isActive: true,
          slot: values.slot,
          responsable: values.responsable,
          type: values.type,
          idResponsable: values.idResponsable,
          licensePlate: values.licensePlate,
        });
        addClients({
          id: 0,
          day: daysOnWeek,
          date: new Date().toISOString(),
          responsable: values.responsable,
          idResponsable: values.idResponsable,
          licensePlate: values.licensePlate,
        });
        setError('');
        resetForm();
      }
    }

    if (values.type === 'Carro') {
      if (permitionInsertStockCar === false) {
        setError('El lugar de carros se encuentra lleno');
      }
      if (permitionInsertStockCar === true && vehicleIsRepeat === false) {
        onSubmit({
          id: 0,
          day: daysOnWeek,
          date: new Date().toISOString(),
          isActive: true,
          slot: values.slot,
          responsable: values.responsable,
          type: values.type,
          idResponsable: values.idResponsable,
          licensePlate: values.licensePlate,
        });
        addClients({
          id: 0,
          day: daysOnWeek,
          date: new Date().toISOString(),
          responsable: values.responsable,
          idResponsable: values.idResponsable,
          licensePlate: values.licensePlate,
        });
        setError('');
        resetForm();
      }
    }

    if (values.type === 'Weight') {
      if (permitionInsertStockWeight === false) {
        setError('El lugar de vehiculos pesados se encuentra lleno');
      }
      if (permitionInsertStockWeight === true && vehicleIsRepeat === false) {
        onSubmit({
          id: 0,
          day: daysOnWeek,
          date: new Date().toISOString(),
          isActive: true,
          slot: values.slot,
          responsable: values.responsable,
          type: values.type,
          idResponsable: values.idResponsable,
          licensePlate: values.licensePlate,
        });
        addClients({
          id: 0,
          day: daysOnWeek,
          date: new Date().toISOString(),
          responsable: values.responsable,
          idResponsable: values.idResponsable,
          licensePlate: values.licensePlate,
        });
        setError('');
        resetForm();
      }
    }
  };
  const formik = useFormik({
    initialValues,
    validationSchema,
    onSubmit: handleSubmit,
  });

  return (
    <form onSubmit={formik.handleSubmit}>
      <h2>{formTitle}</h2>
      <Input
        disabled={disabled}
        name="slot"
        placeholder="Posicion donde va"
        value={formik.values.slot}
        onChange={formik.handleChange}
      />
      {formik.touched.slot && formik.errors.slot && (
        <SpanErrorAlert msg={formik.errors.slot} />
      )}

      <Input
        disabled={disabled}
        name="day"
        placeholder="dia de la semana"
        defaultValue={daysOnWeek}
      />

      <Input
        disabled={disabled}
        name="responsable"
        placeholder="Nombre de el responsable"
        value={formik.values.responsable}
        onChange={formik.handleChange}
      />
      {formik.touched.responsable && formik.errors.responsable && (
        <SpanErrorAlert msg={formik.errors.responsable} />
      )}
      <Select
        disabled={disabled}
        name="type"
        placeholder="Tipo"
        value={formik.values.type}
        onChange={formik.handleChange}
      >
        <option value="">Seleccione</option>
        <option value="Moto">Moto</option>
        <option value="Carro">Carro</option>
        <option value="Weight">Vehiculo pesado</option>
      </Select>
      {formik.touched.type && formik.errors.type && (
        <SpanErrorAlert msg={formik.errors.type} />
      )}

      <Input
        disabled={disabled}
        name="idResponsable"
        placeholder="Identificacion del responsable"
        value={formik.values.idResponsable}
        onChange={formik.handleChange}
      />
      {formik.touched.idResponsable && formik.errors.idResponsable && (
        <SpanErrorAlert msg={formik.errors.idResponsable} />
      )}

      <Input
        disabled={disabled}
        name="licensePlate"
        placeholder="Placa del vehiculo"
        value={formik.values.licensePlate}
        onChange={formik.handleChange}
      />
      {formik.touched.licensePlate && formik.errors.licensePlate && (
        <SpanErrorAlert msg={formik.errors.licensePlate} />
      )}
      {error && <SpanErrorAlert msg={error} />}
      <Button type="submit">Registrar</Button>
    </form>
  );
};

FormCreateVehicle.propTypes = {
  onSubmit: PropTypes.func.isRequired,
  addClients: PropTypes.func.isRequired,
  formTitle: PropTypes.string.isRequired,
  vehicle: PropTypes.array.isRequired,
  disabled: PropTypes.bool,
  initialValues: PropTypes.shape({
    slot: PropTypes.string.isRequired,
    day: PropTypes.string.isRequired,
    responsable: PropTypes.string.isRequired,
    type: PropTypes.string.isRequired,
    idResponsable: PropTypes.string.isRequired,
    licensePlate: PropTypes.string.isRequired,
  }),
};
