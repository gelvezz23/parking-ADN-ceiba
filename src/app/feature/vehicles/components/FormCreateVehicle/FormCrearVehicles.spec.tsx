import * as React from 'react';

import { RenderResult, fireEvent, render, wait } from '@testing-library/react';

import { SinonStub, stub } from 'sinon';
import { Days } from './../../../../shared/components/Days';

import { FormCreateVehicle } from './index';
import { setTextEvent } from './../../../../shared/utils/test';

describe('FormCrearVehicles test', () => {
  let componentWrapper: RenderResult;
  let componentProps: React.ComponentProps<typeof FormCreateVehicle> & {
    onSubmit: SinonStub;
  };
  const twoPosition = 2;
  const treePosition = 3;
  const fourPosition = 4;
  const expectSpanLen = 4;

  beforeEach(() => {
    componentProps = {
      formTitle: 'Form test',
      onSubmit: stub(),
      addClients: stub(),
      vehicle: [],
    };
    componentWrapper = render(<FormCreateVehicle {...componentProps} />);
  });

  it('should match snapshot', () => {
    expect(componentWrapper.container).toMatchSnapshot();
  });

  it('should fail on submit all fields missing', async () => {
    const elem = componentWrapper.container;
    const submitButton = elem.querySelector('button[type="submit"]');

    await wait(() => {
      submitButton && fireEvent.click(submitButton);
    });
    const spans = elem.querySelectorAll('span');
    const spansLen = 5;
    expect(spans.length).toBe(spansLen);
    expect(spans[0].textContent).toBe('Aqui debe ir la posicion de el carro');
    expect(spans[1].textContent).toBe(
      'Debe ir el nombre de el dueño de el vehiculo'
    );
    expect(spans[twoPosition].textContent).toBe(
      'Que tipo de vehiculo es (moto, carro, vehiculo pesado)'
    );
    expect(spans[treePosition].textContent).toBe(
      'Identificacion de el responsable'
    );

    expect(spans[fourPosition].textContent).toBe(
      'Debe ir placa de el vehiculo a registrar'
    );
  });

  it('should fail on submit four fields missing', async () => {
    const elem = componentWrapper.container;
    const responsable = elem.querySelector('input[name="responsable"]');
    const submitButton = elem.querySelector('button[type="submit"]');

    await wait(() => {
      responsable &&
        fireEvent.change(responsable, setTextEvent('responsable', 'Lorem'));
    });

    await wait(() => {
      submitButton && fireEvent.click(submitButton);
    });
    const spans = elem.querySelectorAll('span');
    expect(spans.length).toBe(expectSpanLen);
    expect(spans[0].textContent).toBe('Aqui debe ir la posicion de el carro');
    expect(spans[1].textContent).toBe(
      'Que tipo de vehiculo es (moto, carro, vehiculo pesado)'
    );
  });

  it('should fail on submit four fields missing', async () => {
    const elem = componentWrapper.container;

    const slot = elem.querySelector('input[name="slot"]');
    const day = elem.querySelector('input[name="day"]');
    const submitButton = elem.querySelector('button[type="submit"]');

    await wait(() => {
      slot && fireEvent.change(slot, setTextEvent('slot', 'M1'));
    });
    await wait(() => {
      day && fireEvent.change(day, setTextEvent('day', 'Lunes'));
    });

    await wait(() => {
      submitButton && fireEvent.click(submitButton);
    });
    const spans = elem.querySelectorAll('span');
    expect(spans.length).toBe(expectSpanLen);
    expect(spans[0].textContent).toBe(
      'Debe ir el nombre de el dueño de el vehiculo'
    );
    expect(spans[1].textContent).toBe(
      'Que tipo de vehiculo es (moto, carro, vehiculo pesado)'
    );
    expect(spans[twoPosition].textContent).toBe(
      'Identificacion de el responsable'
    );
    expect(spans[treePosition].textContent).toBe(
      'Debe ir placa de el vehiculo a registrar'
    );
  });

  it('should submit', async () => {
    const elem = componentWrapper.container;

    const slot = elem.querySelector('input[name="slot"]');
    const day = elem.querySelector('input[name="day"]');
    const responsable = elem.querySelector('input[name="responsable"]');
    const type = elem.querySelector('select[name=type]');
    const idResponsable = elem.querySelector('input[name=idResponsable]');
    const licensePlate = elem.querySelector('input[name=licensePlate]');

    const submitButton = elem.querySelector('button[type="submit"]');
    await wait(() => {
      slot && fireEvent.change(slot, setTextEvent('slot', 'slot'));
    });
    await wait(() => {
      day && fireEvent.change(day, setTextEvent('day', 'day'));
    });
    await wait(() => {
      responsable &&
        fireEvent.change(
          responsable,
          setTextEvent('responsable', 'responsable')
        );
    });

    await wait(() => {
      type && fireEvent.change(type, { target: { value: 'Moto' } });
    });

    await wait(() => {
      idResponsable &&
        fireEvent.change(idResponsable, setTextEvent('idResponsable', '1090'));
    });

    await wait(() => {
      licensePlate &&
        fireEvent.change(
          licensePlate,
          setTextEvent('licensePlate', 'licensePlate')
        );
    });

    await wait(() => {
      submitButton && fireEvent.click(submitButton);
    });
    const sundayPosition = 6;
    const date =
      new Date().getDay() === 0 || new Date().getDay() === -1
        ? Days[sundayPosition]
        : Days[new Date().getDay() - 1];
    const formSubmitted = componentProps.onSubmit.firstCall.args[0];

    expect(formSubmitted.slot).toBe('slot');
    expect(formSubmitted.day).toBe(date);
    expect(formSubmitted.responsable).toBe('responsable');
    expect(formSubmitted.type).toBe('Moto');
    expect(formSubmitted.idResponsable).toBe('1090');
    expect(formSubmitted.licensePlate).toBe('licensePlate');
  });
});
