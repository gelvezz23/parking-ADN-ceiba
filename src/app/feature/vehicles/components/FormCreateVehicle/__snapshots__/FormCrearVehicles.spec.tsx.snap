// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormCrearVehicles test should match snapshot 1`] = `
<div>
  <form>
    <h2>
      Form test
    </h2>
    <input
      class="sc-AxiKw deCLXK"
      name="slot"
      placeholder="Posicion donde va"
      value=""
    />
    <input
      class="sc-AxiKw deCLXK"
      name="day"
      placeholder="dia de la semana"
      value="Lunes"
    />
    <input
      class="sc-AxiKw deCLXK"
      name="responsable"
      placeholder="Nombre de el responsable"
      value=""
    />
    <select
      class="sc-AxhCb lmESJy"
      name="type"
      placeholder="Tipo"
    >
      <option>
        Seleccione
      </option>
      <option
        value="Moto"
      >
        Moto
      </option>
      <option
        value="Carro"
      >
        Carro
      </option>
      <option
        value="Weight"
      >
        Vehiculo pesado
      </option>
    </select>
    <input
      class="sc-AxiKw deCLXK"
      name="idResponsable"
      placeholder="Identificacion del responsable"
      value=""
    />
    <input
      class="sc-AxiKw deCLXK"
      name="licensePlate"
      placeholder="Placa del vehiculo"
      value=""
    />
    <button
      class="sc-AxjAm iPvOGw"
      type="submit"
    >
      Registrar
    </button>
  </form>
</div>
`;
